app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+from datetime import date
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Not Vaccinated")
+
+        elif visitor["vaccine"]["expiration_date"] \
+                < date.today():
+
+            raise OutdatedVaccineError("Out dated Vaccine")
+
+        elif visitor["wearing_a_mask"] is False or\
+                "wearing_a_mask" not in visitor:
+
+            raise NotWearingMaskError("Not Wearing Mask")
```

Comment:
add more descriptive messages for the errors

Reply:
fix